### PR TITLE
Add emeraldbgc

### PR DESCRIPTION
**Pull request by SantiagoSanchezF**
Add emeraldbgc
Secondary Metabolite Biosynthetic Gene Clusters (SMBGC) detection tool

### CODE REVIEW

**Comment by SantiagoSanchezF** [comment]
@BiocondaBot please add label

File: recipes/emeraldbgc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "emeraldbgc" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "ad6d2c6ccdca4db504b66c7ac0094acb3bdcf01ae366fce39754cfb7cca08e0e" %}
+
+package:
+
```
Comment (simonbray) [inline recipes/emeraldbgc/meta.yaml:6]:
```suggestion
```

File: recipes/emeraldbgc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - perl >=5
+    - biopython
+    - numpy >=1.16,<1.20
+    - numpy <1.20
```
Comment (simonbray) [inline recipes/emeraldbgc/meta.yaml:30]:
```suggestion
```

File: recipes/emeraldbgc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - openjdk >=11.0
+    - tensorflow =2.4.1
```
Comment (simonbray) [inline recipes/emeraldbgc/meta.yaml:32]: Any chance you can be a bit more flexible with the pinning? Do you really need those exact versions for TF and sklearn?

File: recipes/emeraldbgc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+test:
+  imports:
+    - emeraldbgc
+  commands:
+    - emeraldbgc --help > /dev/null
+
+about:
+  home: "https://github.com/Finn-Lab/emeraldBGC"
+  summary: "SMBGC detection tool"
+  license: "Apache License, Version 2.0"
```
Comment (simonbray) [inline recipes/emeraldbgc/meta.yaml:47]:
```suggestion
  license: Apache-2.0
  license_family: Apache
```

File: recipes/emeraldbgc/meta.yaml (hunk 3 above)
Comment (SantiagoSanchezF) [reply recipes/emeraldbgc/meta.yaml]: I loosened the versions a bit. Bigger changes modify the results

Comment (simonbray) [reply recipes/emeraldbgc/meta.yaml]: Okay :+1:

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
@bioconda-bot please merge

**Pull request merged**